WooCommerce/Classes/ViewRelated/Orders/Order Details/OrderDetailsViewController.swift

```diff
@@ -353,7 +353,14 @@ private extension OrderDetailsViewController {
     /// Presents the order edit form
     ///
     private func editOrder() {
-        // TODO: Implement
+        let viewModel = NewOrderViewModel(siteID: viewModel.order.siteID, initialOrder: viewModel.order)
+        viewModel.onOrderCreated = { [weak self] order in
```

Comment:
Should we change this `onOrderCreated` closure name too?

Reply:
How would you call it? `onChangesCommited`? Editing flow is using this closure mostly as "done" button handler.
We still call "commit all changes" in editing, but don't really need this, right?

Maybe add separated `onOrderCreated` called only on creation flow + `onDoneTapped` on both flows?

Reply:
Maybe an `onFinished` is enough?